### DISCUSSION TITLE
关于“批量下载”的一点建议

### DISCUSSION THREAD

**Discussion by Chlesea** [the1812/Bilibili-Evolved#1544] [功能建议 / Ideas]
## 希望能快速选择32集
> 比如，我选择了第五集，就可以快速的选择5-36集，不需要自己一个一个的去点了
> 或者说，可以自己手动输入选哪几集

 (不知道实现难度怎么样，如果很难做，请大大直接忽略)
 ### 祝大大新年快乐

**Comment by the1812** [comment]
点一下5 按住shift再点一下36
(新年快乐

**Comment by Chlesea** [comment]
解锁新技能，爱了爱了🤣🤣🤣



---原始邮件---
发件人: "Grant Howard"<notifications@github.com&gt;
发送时间: 2021年2月10日(周三) 中午11:45
收件人: "the1812/Bilibili-Evolved"<Bilibili-Evolved@noreply.github.com&gt;;
抄送: "Chlesea"<1556792948@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题: Re: [the1812/Bilibili-Evolved] 关于“批量下载”的一点建议 (#1544)




 
点一下5 按住shift再点一下36
 (新年快乐
